### PR TITLE
Fix broken link in README.md

### LINKED ISSUE #12886: Broken link to Azure DevOps
`markdown-link` failure:
<https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32>

```
[+] ***/tree/master/docs/FAQ.md should work 457ms
2020-06-03T20:01:32.8918939Z       [-] https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work 60.64s
2020-06-03T20:01:32.9327050Z         RuntimeException: Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
2020-06-03T20:01:32.9335025Z         at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
2020-06-03T20:01:33.1943488Z  
```
<https://dev.azure.com/powershell/2972bb5c-f20c-4a60-8bd9-00ffe9987edc/_apis/build/builds/54596/logs/23>

### PR DESCRIPTION
**Pull request by xtqqczze**
# PR Summary

Fix #12886

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ If you have any problems building, please consult the developer [FAQ][].
 
 [FAQ]: https://github.com/PowerShell/PowerShell/tree/master/docs/FAQ.md
 
-[windows-nightly-site]: https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32
+[windows-nightly-site]: https://powershell.visualstudio.com/PowerShell/_build/?definitionId=32
```
Comment (daxian-dbw) [inline README.md:175]:
```suggestion
[windows-nightly-site]: https://powershell.visualstudio.com/PowerShell/_build?definitionId=32
```

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Comment by iSazonov** [comment]
@PoshChan Please rebuild static

**Comment by PoshChan** [comment]
@iSazonov, successfully started rebuild of `PowerShell-CI-static-analysis`

**Comment by iSazonov** [comment]
@xtqqczze It seems another link died. Could you please look?

**Comment by xtqqczze** [comment]
Issue opened: https://github.com/ironmansoftware/universal-dashboard/issues/1652

**Comment by iSazonov** [comment]
@PoshChan Please rebuild static

**Comment by PoshChan** [comment]
@iSazonov, successfully started rebuild of `PowerShell-CI-static-analysis`

**Comment by adityapatwardhan** [comment]
@xtqqczze Thank you for your contribution!

**Pull request merged**